ds248x/ds248x.go

```diff
@@ -315,18 +404,41 @@ var sleep = time.Sleep
 var _ conn.Resource = &Dev{}
 
 const (
-	cmdReset       = 0xf0 // reset ds248x
-	cmdSetReadPtr  = 0xe1 // set the read pointer
-	cmdWriteConfig = 0xd2 // write the device configuration
-	cmdAdjPort     = 0xc3 // adjust 1-wire port
-	cmd1WReset     = 0xb4 // reset the 1-wire bus
-	cmd1WBit       = 0x87 // perform a single-bit transaction on the 1-wire bus
-	cmd1WWrite     = 0xa5 // perform a byte write on the 1-wire bus
-	cmd1WRead      = 0x96 // perform a byte read on the 1-wire bus
-	cmd1WTriplet   = 0x78 // perform a triplet operation (2 bit reads, a bit write)
+	cmdReset         = 0xf0 // reset ds248x
+	cmdSetReadPtr    = 0xe1 // set the read pointer
+	cmdWriteConfig   = 0xd2 // write the device configuration
+	cmdAdjPort       = 0xc3 // adjust 1-wire port (ds2483)
+	cmdChannelSelect = 0xc3 // channel select (ds2482-800)
+	cmd1WReset       = 0xb4 // reset the 1-wire bus
+	cmd1WBit         = 0x87 // perform a single-bit transaction on the 1-wire bus
+	cmd1WWrite       = 0xa5 // perform a byte write on the 1-wire bus
+	cmd1WRead        = 0x96 // perform a byte read on the 1-wire bus
+	cmd1WTriplet     = 0x78 // perform a triplet operation (2 bit reads, a bit write)
 
 	regDCR    = 0xc3 // read ptr for device configuration register
 	regStatus = 0xf0 // read ptr for status register
 	regRDR    = 0xe1 // read ptr for read-data register
 	regPCR    = 0xb4 // read ptr for port configuration register
+	regCSR    = 0xd2 // read ptr for channel selection register
+
+	// ds2482-800 channel selection codes to be written and read back
+	cscIO0w = 0xF0 // channel 0 writing
+	cscIO0r = 0xB8 // channel 0 reading
+	cscIO1w = 0xE1 // channel 1 writing
+	cscIO1r = 0xB1 // channel 1 reading
+	cscIO2w = 0xD2 // channel 2 writing
+	cscIO2r = 0xAA // channel 2 reading
+	cscIO3w = 0xC3 // channel 3 writing
+	cscIO3r = 0xA3 // channel 3 reading
+	cscIO4w = 0xB4 // channel 4 writing
+	cscIO4r = 0x9C // channel 4 reading
+	cscIO5w = 0xA5 // channel 5 writing
+	cscIO5r = 0x95 // channel 5 reading
+	cscIO6w = 0x96 // channel 6 writing
+	cscIO6r = 0x8E // channel 6 reading
+	cscIO7w = 0x87 // channel 7 writing
+	cscIO7r = 0x87 // channel 7 reading
 )
+
+var cscw = []byte{cscIO0w, cscIO1w, cscIO2w, cscIO3w, cscIO4w, cscIO5w, cscIO6w, cscIO7w}
```

Comment:
Use `[...]` so it's an array instead of a slice.
https://blog.golang.org/slices-intro